composer.json

```diff
@@ -12,8 +12,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "ext-mbstring": "*"
+        "php": ">=5.4.0"
```

Comment:
Until PHP 7 is under the "allowed failures" section on Travis, could you use `"php": "^5.4"` please? (also the `>=` is unrecommended by Composer itself).